superset/assets/javascripts/explorev2/components/ChartContainer.jsx

```diff
@@ -149,8 +149,10 @@ class ChartContainer extends React.Component {
     this.props.actions.removeChartAlert();
   }
 
-  renderVis() {
-    visMap[this.props.viz_type](this.state.mockSlice).render();
+  renderVis(heightChange) {
+    if (this.props.chartStatus === 'loading' || heightChange) {
+      visMap[this.props.viz_type](this.state.mockSlice).render();
```

Comment:
there's another method called of`visMap[this.props.viz_type](this.state.mockSlice)` named `resize`. It does assume that `render` has been called prior to `resize` in many cases, meaning you have to persist the object that comes out of `visMap[this.props.viz_type](this.state.mockSlice)`